charts/wa-task-management-api/values.yaml

```diff
@@ -54,8 +54,8 @@ java:
     POSTGRES_REPLICA_HOST: cft-task-postgres-db-flexible-replica-{{ .Values.global.environment }}.postgres.database.azure.com
     POSTGRES_REPLICA_NAME: cft_task_db
     POSTGRES_CONNECTION_OPTIONS: "?sslmode=require"
-    ALLOWED_JURISDICTIONS: wa,ia,sscs,civil,PUBLICLAW,PRIVATELAW,EMPLOYMENT
-    ALLOWED_CASE_TYPES: asylum,wacasetype,sscs,civil,generalapplication,CARE_SUPERVISION_EPO,PRLAPPS,ET_EnglandWales,ET_EnglandWales_Listings,ET_EnglandWales_Multiple,ET_Scotland,ET_Scotland_Listings,ET_Scotland_Multiple,ET_Admin
+    ALLOWED_JURISDICTIONS: wa,ia,sscs,civil,PUBLICLAW,PRIVATELAW,EMPLOYMENT,st_cic
```

Comment:
The intention is that the default config should only match the currently live services.  If this is only intended for `demo` then we only want to apply it in that environment through flux config.

Reply:
Thank you for the information Ravi.